robert_beverage/scripts/generate_qr_code

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env python
 import rospy
+import qrencode
+import argparse
 
 rospy.init_node('generate_qr_code_node')
-rospy.loginfo('Inside generate_qr_code script')
\ No newline at end of file
+rospy.loginfo('Inside generate_qr_code script')
+
+parser = argparse.ArgumentParser(description='generate qr-code')
+parser.add_argument('data', metavar='D', type=str, nargs=5)
```

Comment:
please add separate flag for each option x, y, host, port and filename not single flag DATA :)